bin/ChakraCore/ChakraCore.def

```diff
@@ -38,3 +38,9 @@ JsTTDNotifyYield
 JsTTDPrepContextsForTopLevelEventMove
 JsTTDMoveToTopLevelEvent
 JsTTDReplayExecution
+
+JsInitializeModuleRecord
+JsParseModuleSource
+JsModuleEvaluation
+JsSetModuleHostInfo
+JsGetModuleHostInfo
```

Comment:
Add this issue to track this https://github.com/Microsoft/ChakraCore/issues/1269